src/lib/wp-cli-process.ts

```diff
@@ -11,7 +11,7 @@ export type MessageName = 'execute';
 export type WpCliResult = ReturnType< typeof executeWPCli >;
 export type MessageCanceled = { error: Error; canceled: boolean };
 
-const DEFAULT_RESPONSE_TIMEOUT = 120000;
+const DEFAULT_RESPONSE_TIMEOUT = 180000;
```

Comment:
Could we make it more readable? For example:

```suggestion
const DEFAULT_RESPONSE_TIMEOUT_MS = 180 * 1000;
```

Reply:
Updated in https://github.com/Automattic/studio/pull/741/commits/b9cd537ede2e40e8fde2177de44ecd9263056d16

Reply:
Thanks! Do you see anything against adding `_MS`? We follow that practice in Calypso, too.